src/webrtc/call.ts

```diff
@@ -1213,6 +1309,44 @@ export class MatrixCall extends EventEmitter {
         }
     };
 
+    private async gotCallFeedsForAnswer(callFeeds: CallFeed[]): Promise<void> {
```

Comment:
I don't think it makes sense to add new callFeedSpecific methods with all of this duplicate code. We should DRY this up.

Reply:
Yeah, I definitely agree, will try to do that next time I interact with the code

Reply:
Continued in https://github.com/matrix-org/matrix-js-sdk/pull/1955